server/src/main/java/org/opensearch/action/admin/indices/create/CreateIndexRequest.java

```diff
@@ -483,33 +481,4 @@ public void writeTo(StreamOutput out) throws IOException {
         }
         waitForActiveShards.writeTo(out);
     }
-
-    @Override
-    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
```

Comment:
Why we are removing `ToXContentObject`?

Reply:
Because it's no longer used by the HLRC. Those methods (only used in IndicesClient) were deprecated long ago and are being removed in this PR. Some have already been removed.

e.g., 
```
     * @deprecated This method uses old request and response objects which still refer to types, a deprecated feature.
     * The method {@link #getFieldMapping(GetFieldMappingsRequest, RequestOptions)} should be used instead, which
     * accepts a new request object.
```